src/containers/Flow/FlowList/FlowList.module.css

```
@@ -7,6 +7,12 @@
   height: 48px;
 }
 
+.customAutocomplete {
```

```suggestion
.CustomAutocomplete {
```

classnames should always be capital